gui/screens/settings_wallet.py

```diff
@@ -38,14 +40,25 @@ def __init__(self):
     @allure.step('Open add account pop up in wallet settings')
     def open_add_account_pop_up(self):
         self._wallet_settings_add_new_account_button.click()
-        return AccountPopup().wait_until_appears()
+        return AccountPopup()
```

Comment:
Can we replace it with waiting for some element of AccountPopup? So in case this popup doesn't appear it will fail with error

Reply:
i think we can do the way we do in other places - add try / except , wdyt? @Valentina1133 
i also can improve init of AccountPopup to check if its header lets say is present

Reply:
Yes, great ideas!

Reply:
@Valentina1133 added those, pls check that out